### PR TITLE
update dev dependencies

### PR DESCRIPTION
**Pull request by kukushking**
*Description of changes:*
- update dev dependencies to address [vulnerability](https://github.com/awslabs/aws-codeseeder/security/dependabot/1)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: requirements-dev.in [hunk 1]
```diff
@@ -16,6 +16,7 @@ types-setuptools~=57.4.0
 changelog-cli~=0.7.0
 twine~=5.1.0
 sphinx~=4.3.0
-moto[all]~=4.0.9
+moto[all]>=4.0.13
+networkx~=3.1.0
```
Comment (kukushking) [inline requirements-dev.in:20]: had to force networkx here because older versions don't support py38

**Review by malachi-constant — APPROVED** [review]

**Review by dgraeber — COMMENTED** [review]
@kukushking I see the pydantic version was updated.  Did you perchance test this version in a deployment with SF?

**Comment by kukushking** [comment]
Nope, but this is dev dependencies - shouldn't affect the package

**Comment by kukushking** [comment]
I'll give it a try though

**Review by dgraeber — APPROVED** [review]

**Pull request merged**